src/foundry/client-esm/canvas/sources/point-light-source.d.mts

```diff
@@ -0,0 +1,46 @@
+import BaseLightSource from "./base-light-source.mts";
+import type { PointEffectSourceMixin_BaseLightSource_Interface } from "./point-effect-source.d.mts";
+import type PointEffectSourceMixin from "./point-effect-source.d.mts";
+
+declare const MixedPointBaseLightSource: PointEffectSourceMixin_BaseLightSource_Interface;
+
+type LightSourceData = PointEffectSourceMixin.PointEffectSourceData & BaseLightSource.LightSourceData;
+
+/**
+ * A specialized subclass of the BaseLightSource which renders a source of light as a point-based effect.
+ */
+export default class PointLightSource<
+  SourceData extends LightSourceData = LightSourceData,
+  SourceShape extends PointSourcePolygon = PointSourcePolygon,
+> extends MixedPointBaseLightSource<SourceData, SourceShape> {
+  /** @defaultValue `"lightSources"` */
+  static override effectsCollection: string;
+
+  override _initialize(data: Partial<SourceData>): void;
+
+  override _createShapes(): void;
+
+  override _configure(changes: Partial<SourceData>): void;
+
+  override _getPolygonConfiguration(): PointSourcePolygonConfig;
+
+  /**
+   * Test whether this LightSource provides visibility to see a certain target object.
+   * @param config - The visibility test configuration
+   * @returns Is the target object visible to this source?
+   */
+  testVisibility(config?: {
+    /** The sequence of tests to perform */
+    tests: CanvasVisibilityTest[];
+    /** The target object being tested */
+    object: PlaceableObject;
+  }): boolean;
```

Comment:
This is saying you can only do either `testVisibility()` or `testVisibility({ tests: [...], object: ... })` is that correct?

You CAN'T do `testVisibility({ tests: [...] })` or `testVisibility({ object: ... })` with this type which I find a bit odd.

Reply:
Looks like the optional marker on config is incorrect, both properties are always required.